src/dist/replication/lib/backup/replica_backup_manager.cpp

```diff
@@ -44,6 +44,15 @@ static bool get_policy_checkpoint_dirs(const std::string &dir,
     return true;
 }
 
+replica_backup_manager::replica_backup_manager(replica *r) : replica_base(r), _replica(r) {}
+
+replica_backup_manager::~replica_backup_manager()
+{
+    if (_collect_info_timer != nullptr) {
+        _collect_info_timer->cancel(true);
```

Comment:
Will `cancel` timer guarantee `_collect_info_timer = nullptr`, so you remove it?

Reply:
the destructor of replica_backup_manager will guarantee  `_collect_info_timer = nullptr`, not cancel